api/datastore/sql/migrations/15_alter_apps.go

```diff
@@ -0,0 +1,84 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/fnproject/fn/api/datastore/sql/migratex"
+	"github.com/jmoiron/sqlx"
+)
+
+func up15(ctx context.Context, tx *sqlx.Tx) error {
+	_, err := tx.ExecContext(ctx, "ALTER TABLE apps RENAME TO old_apps;")
+	if err != nil {
+		return err
+	}
+	newTable := `CREATE TABLE apps (
+	id varchar(256) NOT NULL PRIMARY KEY,
+	name varchar(256) NOT NULL UNIQUE,
+	config text NOT NULL,
+	annotations text NOT NULL,
+	created_at varchar(256),
```

Comment:
Nit pick, but these should be NOT NULL to right?

Reply:
Perhaps, but that's unrelated if so.

Reply:
ah, good idea

Reply:
there may exist some fns that actually do have apps with a null created_at, since it was an addition a while ago (last november?), but it seems unlikely